README.md

```diff
@@ -137,6 +137,22 @@ yarn install
 yarn test
 ```
 
+#### Generating Test Coverage Report
+
+To generate a test coverage report using Jest, follow these steps:
```

Comment:
Do you think contributors would run it if it's optional?  What if we add it to the pre-commit hook?

https://github.com/OpenBeta/sandbag/blob/develop/.husky/pre-commit

Reply:
I could add it if you'd prefer! It takes 20ish seconds to generate the report though, so running the tests could be enough pre-commit. Those interested could then run it adhoc.